### ISSUE TITLE
Keyboard navigation does not work for Continuous Toolbox

### ISSUE THREAD

**Issue by alschmiedt** [RaspberryPiFoundation/blockly-samples#399]
**Category**

**Component**
Continuous Toolbox

**Describe the bug**
- Upon entering keyboard navigation mode, the flyout closes. 
- When you try to focus on the flyout it will always put the cursor around the first item in the flyout

**To Reproduce**
- Open continuous toolbox
- Enter keyboard navigation mode. (ctrl + shift + k)

**Expected behavior**
- Keyboard navigation should not call hide on the flyout if `autoClose` is false.
- The cursor should focus on the first block in a category, instead of always focusing on the first block in the flyout. (ex: If we are on the Loops category the cursor should go around the first Loops block)

**Screenshots**

**Additional context**
Since I am planning on working on keyboard nav this quarter, I will take a look at this after I have moved keyboard navigation to a plugin.

**Comment by gonfunko** [comment]
Closing since this plugin has been removed.
